adapters/core2p2p/transaction.go

```diff
@@ -97,10 +96,9 @@ func AdaptTransaction(transaction core.Transaction) *spec.Transaction {
 					L1Gas:             adaptResourceLimits(tx.ResourceBounds[core.ResourceL1Gas]),
 					L2Gas:             adaptResourceLimits(tx.ResourceBounds[core.ResourceL2Gas]),
 					Tip:               AdaptFelt(new(felt.Felt).SetUint64(tx.Tip)),
-					// todo fill rest of V3 fields:
-					Paymaster:   nil,
-					NonceDomain: "",
-					FeeDomain:   "",
+					Paymaster:         nil,
```

Comment:
Paymaster is some kind of obsolete field?

Reply:
I am not sure, since it was both in core and p2p spec but their types were different I left it as nil.